packages/sdk-rtl/src/errorDoc.spec.ts

```diff
@@ -222,10 +222,7 @@ describe('ErrorDoc', () => {
       ['login_2_404.md', 'login_2'],
       ['and_another_404.md', 'and_another'],
       ['errorcodes/live/login_404.md', 'login'],
-      [
-        'https://marketplace-api.looker.com/errorcodes/live/login_404.md',
-        'login',
-      ],
+      ['https://static-a.cdn.looker.app/errorcodes/live/login_404.md', 'login'],
```

Comment:
Just remove /live from this path though. https://static-a.cdn.looker.app/errorcodes/login_404.md works!

Reply:
FWIW the `/live` path could have been anything. I was testing to make sure localized error docs would also resolve to a method name.